toolchain/sem_ir/function.h

```diff
@@ -23,6 +23,12 @@ struct FunctionFields {
   // always present if the function has a declared return type.
   InstId return_storage_id;
 
+  enum class VirtualModifier { None, Virtual, Abstract, Impl };
```

Comment:
Sorry, can you move this to the top (for [declaration order styles](https://google.github.io/styleguide/cppguide.html#Declaration_Order), types/enums come before  instance members)

Reply:
Ah, sure - done in https://github.com/carbon-language/carbon-lang/pull/4272/commits/a2d80a6bffeee49e89fdcccfdee812c012b505e2